src/content/configuration/dev-server.md

```diff
@@ -749,7 +749,7 @@ module.exports = {
 Usage via the CLI
 
 ```bash
-npx webpack serve --no-live-reload
+npx webpack serve --liveReload
 ```
 
```

Comment:
```suggestion
Notice that there's no way to disable it from CLI.

```
I would suggest adding a note below as the cli example is added right after the example with `liveReload` set to `false`.

Reply:
Makes sense 👍🏻 .